views/Admin.react.js

```diff
@@ -696,6 +765,27 @@ class AdminPage extends Component {
                 dataKey="access"
                 width={this.state.width / 6}
               />
+              <Column
+                label="Account Expiration"
+                cellRenderer={({ cellData, rowData: { uid } }) => (
+                  <TextField
+                    id="account_expires"
+                    name="account_expires"
+                    type="date"
+                    defaultValue={moment(cellData).format('YYYY-MM-DD')}
+                    InputLabelProps={{
+                      shrink: true,
+                    }}
+                    onChange={(e) =>
+                      this.updatePermissions(uid, {
+                        account_expires: e.target.value,
+                      })
+                    }
```

Comment:
the nice thing about having the name and value on the form element (besides being semantic HTML) is that this allows us to use a generic method here, something like:

```js
onChange={(e) => this.updateUser(uid, { [e.target.name]: e.target.value })}
```

We can then use the `this.updateUser` on any form element changing a user attribute.

Reply:
👍 [8814b2b](https://github.com/AMP-SCZ/dpdash/pull/294/commits/8814b2b15da0a94568320a77593fb7c635f9ffb9)